data/json/terrain.json

```diff
@@ -29,14 +28,214 @@
+        "color": "brown",
+        "move_cost": 0,
+        "flags": ["NOITEM", "SUPPORTS_ROOF", "WALL", "AUTO_WALL_SYMBOL"],
+        "connects_to": "WALL",
+        "bash": {
+            "str_min": 10, "str_max": 80,
+            "sound": "crash!",
+            "sound_fail": "bash!",
+            "ter_set": "t_wf_dmg",
+            "items": [
+                { "item": "rock", "count": [5, 10] },
+				{ "item": "splinter", "count": [2, 3] }
+            ]
+        }
+    },{
+        "type" : "terrain",
+        "id" : "t_wf_dmg",
+        "name": "damaged wood building frame",
+        "description": "",
+        "symbol": "LINE_OXOX",
+        "color": "brown",
+        "move_cost": 0,
+        "flags": ["NOITEM", "SUPPORTS_ROOF", "WALL", "AUTO_WALL_SYMBOL"],
+        "connects_to": "WALL",
+        "bash": {
+            "str_min": 10, "str_max": 80,
+            "sound": "crash!",
+            "sound_fail": "bash!",
+            "ter_set": "t_wf_dmg",
+            "items": [
+                { "item": "rock", "count": [5, 10] },
+				{ "item": "splinter", "count": [2, 3] }
+            ]
+        }
     },{
+        "type" : "terrain",
+        "id" : "t_df_wreck",
+        "name": "wrecked wood building frame",
+        "description": "",
+        "symbol": "LINE_OXOX",
+        "color": "brown",
+        "move_cost": 0,
+        "flags": ["NOITEM", "SUPPORTS_ROOF", "WALL", "AUTO_WALL_SYMBOL"],
+        "connects_to": "WALL",
+        "bash": {
+            "str_min": 10, "str_max": 80,
+            "sound": "crash!",
+            "sound_fail": "bash!",
+            "ter_set": "t_wf_dmg",
+            "items": [
+                { "item": "rock", "count": [5, 10] },
+				{ "item": "splinter", "count": [2, 3] }
+            ]
+        }
+    },{
+        "type" : "terrain",
+        "id" : "t_brick_wfwall",
+        "name": "framed brick wall",
+        "description": "A frame construction wall lined with bricks for apearence rather than structure.  Popular in modern construction it's not as tough as a true brick wall but is more than enough to keep out the casual intruder.",
+        "symbol": "LINE_OXOX",
+        "color": "brown",
+        "move_cost": 0,
+        "flags": ["NOITEM", "SUPPORTS_ROOF", "WALL", "AUTO_WALL_SYMBOL"],
+        "connects_to": "WALL",
+        "bash": {
+            "str_min": 40, "str_max": 140,
+            "sound": "crash!",
+            "sound_fail": "bash!",
+            "ter_set": "t_null",
+            "items": [
+                { "item": "rock", "count": [5, 10] },
+				{ "item": "splinter", "count": [2, 3] }
+            ]
+        }
+    },{
+        "type" : "terrain",
+        "id" : "t_conc_sfwall",
+        "name": "wall - concrete framed",
+        "description": "A steel frame with concrete commonly used in comercial construction.  While thinner and thus less fortied than a true concrete wall, the frames will continue to provied structural integraty should the wall get destroyed.",
```

Comment:
Commercial, fortified, provide, integrity.

Reply:
Its not very clear what this is supposed to be, is it a steel column and a non-structural precast concrete panel wall in the same tile?

Or is the steel reinforcement inside the wall, like in a  concrete shear wall?

Reply:
I had like big box store or office building in mind...